NitroxClient/MonoBehaviours/Gui/MainMenu/ConsoleJoinServer.cs

```diff
@@ -92,6 +97,25 @@ public void OnConsoleCommand_warpto(NotificationCenter.Notification n)
             }
         }
 
+        public void OnConsoleCommand_sub(NotificationCenter.Notification n)
```

Comment:
That is cool, I didn't know you could have two handlers to the dev console events.